src/code/code_800BB0A0.c

```diff
@@ -1,5 +1,73 @@
 #include "global.h"
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/code_800BB0A0/func_800BB0A0.s")
+/**
+ * Cutscene camera movement carried over from Super Mario 64. Unused in the release game.
```

Comment:
the bottom function is also carried over, so it feels weird to single out this one specifically. kinda think it would be good as a comment at the top of the file

Reply:
i put this at the top of the file
`// These two functions are very similar to a spline system used for cutscene camera movement in Super Mario 64`

in that game it is actually just used for cutscenes camera movement